README.md

```diff
@@ -176,6 +183,72 @@ req_url_path("study", my_study_id, "patient") |>
 
 This endpoint determines the patient's treatment group.
 
+### Study List
+
+The GET /study/ endpoint allow to list all previously defined studies. It returns information such as:
+
+- Study ID
+- Identifier
+- Name of study
+- Randomization method
+- Last edit date
+
+
+```R
+# Print all defined studies
+req_url_path("study", "") |>
+  req_method("GET") |>
+  req_perform()
+```
```

Comment:
I think:
- if this is not executed, then brings no value and would vote removing
- if this is executed, then we need to take care payload is presented properly